cartridge/utils.lua

```diff
@@ -305,6 +306,33 @@ local function http_read_body(req)
     return form_body
 end
 
+--- Wait for box.info.vclock[id] to reach desired LSN.
+--
+-- @function wait_lsn
+-- @local
+--
+-- @tparam number id
+-- @tparam number lsn
+-- @tparam number pause
+-- @tparam number timeout
+-- @treturn boolean true / false
+local function wait_lsn(id, lsn, pause, timeout)
+    checks('number', 'number', 'number', 'number')
+    local deadline = fiber.time() + timeout
```

Comment:
I suppose usage of `require('clock').monotonic()` is better when you work with timeouts.

Reply:
I've created new follow-up issue: https://github.com/tarantool/cartridge/issues/957

`fiber.time` is common for cartridge, so I'd rather resolve it aside.